### DISCUSSION TITLE
Middle Mouse button stopped working on detail and edit links on index.

### DISCUSSION THREAD

**Discussion by andypooletrioteca** [laravel/nova-issues#6081] [Ideas & Feature Requests] [answered]
- Laravel Version: 10.x
- Nova Version: 4.32.6
- PHP Version: 8.2
- Database Driver & Version:
- Operating System and Version: any
- Browser type and version: any

### Description:
Clicking the middle mouse button won't open the link in a new tab for the detail and edit buttons on the index resource.

### Detailed steps to reproduce the issue on a fresh Nova installation:
open a index page with some resources and click the middle mouse button on a detail or edit button of a resource.

**Comment by davidhemphill** [comment]
Nova has never had event handlers for middle click.

**Reply by dmason30** [reply]
@davidhemphill This definitely used to work because the buttons were rendered as links. I assume the recent changes now render them as buttons?

**Reply by MPaap** [reply]
It's default browser behavior to open a new tab on click of the middle mouse button.
Also right mouse button -> open in new tab does not work properly, it just opens the current page.
Control + Click opens 2 tabs, one that is the current page and the other opens the resource what is the expected behavior.

**Comment by crynobone** [comment, marked as answer]
Fixed in v4.32.14
